app/assets/stylesheets/stream_element.scss

```diff
@@ -72,15 +107,25 @@
 }
 
 .stream-element {
-  background-color: $framed-background;
   padding: 10px;
 
+  .post-wrapper {
+    border: 1px solid #ddd;
+    border-radius: 3px;
+    background-color: #f9f8f8;
+  }
+
+  .post-content-wrapper {
+    padding: 10px;
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.